### PR TITLE
Add primary action to high priority tasks

### PR DESCRIPTION
**Pull request by octaedro**
This is a follow-up PR for [PR 2787](https://github.com/Automattic/woocommerce-payments/pull/2787) to address the modifications suggested in [this comment](https://github.com/Automattic/woocommerce-payments/issues/2455#issuecomment-895503181).

#### Changes proposed in this Pull Request

This PR adds a primary action for the tasks: `Reconnect WooCommerce Payments` and `Update WooCommerce Payments business details`.

Now the task `Reconnect WooCommerce Payments` will have a `Reconnect` button

screenshot-localhost_8082-2021 09 15-13_03_55

and the task `Update WooCommerce Payments business details` will have an `Update` button.

screenshot-localhost_8082-2021 09 15-13_03_33

#### Testing instructions

* The easiest way to test this PR is using this branch `add/primary_action_to_high_priority_tasks_test`.
* The long and safer way to test this PR is by following the instructions described [here](https://github.com/Automattic/woocommerce-payments/pull/1799#issue-644251556) to test the task `Update WooCommerce Payments business details` and [these instructions](https://github.com/Automattic/woocommerce-payments/pull/1912#issue-902593024) to test the task `Reconnect WooCommerce Payments`.

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)

### CODE REVIEW

**Comment by joelclimbsthings** [comment]
Hey @octaedro , I haven't tested in full just yet, but it looks like these modifications are solely to the soon-to-be-obsolete tasks files. We're close to deleting the logic within `/client/task-list` in favor of the new Tasks API implemented in `/client/tasks`. 

Would it be possible to implement this in the new tasks before we spend time on testing?

**Comment by octaedro** [comment]
Hi @joelclimbsthings,

I'm not sure if I understood your message correctly, is the `Payments Overview` tasklist going to be deprecated?

>  We're close to deleting the logic within /client/task-list in favor of the new Tasks API implemented in /client/tasks.

I couldn't find the folder you mentioned in this repo, are you sure you're referring to the `Overview` tasklist?

**Comment by joelclimbsthings** [comment]
Sorry @octaedro, false alarm. Made a incorrect assumption at first glance. I'll take a closer look at this shortly!

File: client/overview/task-list/tasks.js [hunk 1]
```diff
@@ -77,14 +77,18 @@ export const getTasks = ( {
 				),
 				additionalInfo: accountDetailsTaskDescription,
```
Comment (louwie17) [inline client/overview/task-list/tasks.js:78]: I think we should change this to `content` given it generates a bigger gap between the title and the extra text, see screenshot:
Screen Shot 2021-09-23 at 11 41 47 AM

File: client/overview/task-list/tasks.js [hunk 2]
```diff
@@ -100,9 +104,13 @@ export const getTasks = ( {
 					'woocommerce-payments'
```
Comment (louwie17) [inline client/overview/task-list/tasks.js:104]: Same as above, use `content` instead of `additionalInfo` on line 101 here.

**Review by louwie17 — COMMENTED** [review]
Tested well, but made two suggestions, using `content` versus `additionalInfo`, see the screenshot in the comments for the difference.

File: client/overview/task-list/tasks.js (hunk 1 above)
Comment (octaedro) [reply client/overview/task-list/tasks.js]: Nice catch! I addressed that change in the commit aed0044

**Comment by octaedro** [comment]
Thank you @louwie17 for your feedback.
I addressed the suggested changes. Could you give this PR another look?

**Review by louwie17 — APPROVED** [review]
Looks good now, thanks for the updates @octaedro, this is ready to 🚢

**Pull request merged**